src/System.Windows.Forms/src/System/Windows/Forms/MonthCalendar.CalendarBodyAccessibleObject.cs

```diff
@@ -10,66 +14,216 @@ namespace System.Windows.Forms
     public partial class MonthCalendar
     {
         /// <summary>
-        /// Represents the calendar body accessible object.
+        ///  Represents a calendar body accessible object of a MonthCalendar control.
```

Comment:
What's a "calendar body"?
```suggestion
        ///  Represents an accessible object for a calendar body in <see cref="MonthCalendar"/> control.
```